docker/php/Dockerfile

```diff
@@ -5,6 +5,8 @@ RUN apt-get update && apt-get install -y \
     && docker-php-ext-install \
         iconv mcrypt mbstring intl pdo pdo_mysql pdo_pgsql
 
+RUN echo "date.timezone=Europe/Paris" > /usr/local/etc/php/conf.d/date_timezone.ini
```

Comment:
Maybe this could rely on a [build argument](https://docs.docker.com/engine/reference/builder/#arg), so it will be overridable in `docker-compose.yml` without change in `Dockerfile` ? (starting with *docker-compose* 1.6)

Is there some non-"french timezoned" developers on Wallabag ?